tests/stdlib/tnet_ll.nim

```diff
@@ -29,7 +29,7 @@ suite "inet_ntop tests":
     check: ip4.s_addr == 0x10111213'u32
 
     var buff: array[0..255, char]
-    let r = inet_ntop(AF_INET, cast[pointer](ip4.s_addr.addr), buff[0].addr, buff.len.int32)
+    let r = inet_ntop(AF_INET, cast[pointer](ip4.s_addr.addr), cast[cstring](buff[0].addr), buff.len.int32)
     let res = if r == nil: "" else: $r
```

Comment:
`addr buffer` vs `addr buffer[0]`

Reply:
what's the (practical) difference?

Reply:
I don't know its practical difference. Pre-existing.

Reply:
`addr buffer[0]` also works for strings and sequences.